### ISSUE TITLE
Test Data Generator

### ISSUE THREAD

**Issue by ashokdudhat** [wpeventmanager/wp-event-manager#242]
https://theeventscalendar.com/extensions/elegant-themes-divi-compatibility/

New plugin or extension.

**Comment by hiteshmakvana** [comment]
@krinaydhanani Can you please prepare sample data

**Comment by krinaydhanani** [comment]
@hiteshmakvana Yes sure can you give me the list of how many data you want. Like a number of Events, Organizer, Venue, Registration every thing.
